app/Exceptions/Handler.php

```diff
@@ -36,14 +35,12 @@ class Handler extends ExceptionHandler
     ];
 
     /**
-     * Report or log an exception.
-     *
-     * @param  \Throwable  $e
-     * @return void
+     * Register the exception handling callbacks for the application.
      */
-    public function report(Throwable $e)
+    public function register()
```

Comment:
We talked about this; we just wanted to check that this will behave in the same way as before but is the new recommended way to do this.

Reply:
Looking at the [docs](https://laravel.com/docs/10.x/errors) this seems to be recommended, although I now realized a reason to override the report function may be if you want to call it yourself when there is no Exception (see section [The report helper](https://laravel.com/docs/10.x/errors#the-report-helper))

Reply:
I don't think we do this anywhere though

Reply:
Okay I have to correct myself again, after testing locally I could verify that the handler still gets called this way, so we should be good.